app/src/end2endTest/resources/test-6602-01/assessment.json

```diff
@@ -150,5 +150,6 @@
     "relevantMedCount": 4,
     "totalMedCount": 11
   },
-  "claimSubmissionId": "7002"
+  "claimSubmissionId": "7002",
+  "sufficientForFastTracking": false
```

Comment:
Julian fixed this back so that sufficientForFastTracking is not introduced. I am not sure why we are putting this back. I am not against adding is but it should not be false and MAS should not change the V1 assessor functionality to my understanding.

Reply:
This field should be required, even if you ignore it for V1 processing. If the field is missing, it should default to false. The functionality for V1 does not change, since the field is simply ignored.